src/screen.c

```diff
@@ -3724,6 +3724,16 @@ win_line(
 		    if ((wp->w_p_cul || wp->w_p_rnu)
 						 && lnum == wp->w_cursor.lnum)
 			char_attr = hl_attr(HLF_CLN);
+
+                    if (wp->w_p_rnu && (lnum < wp->w_cursor.lnum))
+                        /* high light relative above (<) cursor */
+                        char_attr = hl_combine_attr(
+                                char_attr, hl_attr(HLF_RNA));
+
+                    if (wp->w_p_rnu && (lnum > wp->w_cursor.lnum))
+                        /* high light relative above (<) cursor */
```

Comment:
Taryn,
   Thank you.  I have added another commit to fix this obvious issue.  I
can leave things as they are, fixup the branch as it sits (rewriting
history), or submit a new PR.  Let me know what the project prefers.

Thanks!

SB

On Thu, Feb 4, 2016 at 11:27 PM, Taryn Hill <vim-dev-github@256bit.org>
wrote:

> In src/screen.c <https://github.com/vim/vim/pull/624#discussion_r51976064>
> :
>
> > @@ -3724,6 +3724,16 @@ win_line(
> >  		    if ((wp->w_p_cul || wp->w_p_rnu)
> >  						 && lnum == wp->w_cursor.lnum)
> >  			char_attr = hl_attr(HLF_CLN);
> > +
> > +                    if (wp->w_p_rnu && (lnum < wp->w_cursor.lnum))
> > +                        /* high light relative above (<) cursor */
> > +                        char_attr = hl_combine_attr(
> > +                                char_attr, hl_attr(HLF_RNA));
> > +
> > +                    if (wp->w_p_rnu && (lnum > wp->w_cursor.lnum))
> > +                        /* high light relative above (<) cursor */
>
> This comment is exactly the same as the above comment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vim/vim/pull/624/files#r51976064>.
>
> --
> --
> You received this message from the "vim_dev" maillist.
> Do not top-post! Type your reply below the text you are replying to.
> For more information, visit http://www.vim.org/maillist.php
>
> ---
> You received this message because you are subscribed to the Google Groups
> "vim_dev" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to vim_dev+unsubscribe@googlegroups.com.
> For more options, visit https://groups.google.com/d/optout.
>